src/modules/countComments.js

```diff
@@ -0,0 +1,3 @@
+const totalComments = (commentsArray) => commentsArray.length;
```

Comment:
[OPTIONAL] Unfortunately, the previous reviewer didn't notice. So I will ask for it as optional. According to the requirements, this function needs to count the element visible, not the data provided by API. Currently, you are getting the data from API. It same for totalMeals function.
<img width="1082" alt="Screenshot 2023-03-17 at 10 15 22 PM" src="https://user-images.githubusercontent.com/25887253/225999494-6e081ca6-0a04-43ad-9262-23741cd560a8.png">